client/extensions/hello-dolly/hello-dolly-page.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+import getLyric from './state/selectors';
```

Comment:
I find it helps to use the same name on the `import` as the module exports - i.e. it helps when searching the code base and it keeps one less than on the programmer's mental stack. Would you mind changing `getLyric` to `getCurrentLyric`?